test/screenshot/golden.json

```diff
@@ -1399,6 +1399,14 @@
       "desktop_windows_ie@11": "https://storage.googleapis.com/mdc-web-screenshot-tests/abhiomkar/2019/11/01/18_25_15_143/spec/mdc-textfield/classes/baseline.html.windows_ie_11.png"
     }
   },
+  "spec/mdc-textfield/classes/disabled-character-counter.html": {
```

Comment:
Let's delete this file too since this is already covered in below screenshot.

Reply:
It's a baseline test that was missing before and failed to catch that the character counter color was previously incorrect when disabled. It's not one of the mixin tests